### PR TITLE
DW: Add test coverage for custom ROCm Ray image

### PR DESCRIPTION
**Pull request by ChughShilpa**
Closes [RHOAIENG-12484](https://issues.redhat.com/browse/RHOAIENG-12484)

### CODE REVIEW

**Review by jiripetrlik — APPROVED** [review]

**Review by abhijeet-dhumal — COMMENTED** [review]
/lgtm

**Comment by ChughShilpa** [comment]
Test passed in `rhoai-test-flow/2044`

**Review by sutaakar — APPROVED** [review]

File: ods_ci/tests/Tests/0600__distributed_workloads/test-run-distributed-workloads-tests.robot [hunk 1]
```diff
@@ -55,21 +55,41 @@ Run TestRayTuneHPOGpu ODH test with Python 3.11
...
     ...     DistributedWorkloads
     ...     Training
     ...     WorkloadsOrchestration
-    Run DistributedWorkloads ODH Test    TestMnistCustomRayImageGpu    ${RAY_TORCH_CUDA_IMAGE_3.11}    ${NOTEBOOK_IMAGE_3.11}
+    Run DistributedWorkloads ODH Test    TestMnistCustomRayCudaGpu    ${RAY_TORCH_CUDA_IMAGE_3.11}    ${NOTEBOOK_IMAGE_3.11}
+
+Run TestKueueCustomRayRocmCpu ODH test with Python 3.11
+    [Documentation]    Run Go ODH test: TestKueueCustomRayRocmCpu
+    [Tags]  RHOAIENG-12484
+    ...     ROCm
```
Comment (sutaakar) [inline ods_ci/tests/Tests/0600__distributed_workloads/test-run-distributed-workloads-tests.robot:80]: Makes me think, this test runs on CPU so ROCm flag is not needed, right?

Comment (ChughShilpa) [reply ods_ci/tests/Tests/0600__distributed_workloads/test-run-distributed-workloads-tests.robot:80]: Yes right. I will raise another PR to remove this tag

Comment (ChughShilpa) [reply ods_ci/tests/Tests/0600__distributed_workloads/test-run-distributed-workloads-tests.robot:80]: Raised PR - https://github.com/red-hat-data-services/ods-ci/pull/2127

**Pull request merged**